src/net_processing.cpp

```diff
@@ -1707,8 +1707,10 @@ void PeerManagerImpl::RelayAddress(NodeId originator,
     // Relay to a limited number of other nodes
     // Use deterministic randomness to send to the same nodes for 24 hours
     // at a time so the m_addr_knowns of the chosen nodes prevent repeats
-    uint64_t hashAddr = addr.GetHash();
-    const CSipHasher hasher = m_connman.GetDeterministicRandomizer(RANDOMIZER_ID_ADDRESS_RELAY).Write(hashAddr << 32).Write((GetTime() + hashAddr) / (24 * 60 * 60));
+    const uint64_t hash_addr = CServiceHash(0, 0)(addr);
```

Comment:
b3a1a3ba A comment documenting why (0, 0) is passed to CServiceHash may be nice (or perhaps a dedicated/named ctor).

Reply:
The comment above says `// Use deterministic randomness ...`. I think that is sufficient but would be happy to reword the existent comment if you have a suggestion.